mscore/editstafftype.cpp

```diff
@@ -485,12 +485,14 @@ void EditStaffType::tabStemThroughCompatibility(bool checked)
 void EditStaffType::updatePreview()
       {
       setFromDlg();
-      if (preview) {
+      if (preview && preview->score()) {
```

Comment:
Whatever Werner said. Our vision of defensive programming is notoriously different :)